html/captive-portal/templates/server_error.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Fatal error !</title>
+  </head>
+
+  <body>
+    <h2>Fatal error !</h2>
+    <p>A server error has occured, please try again or try to contact your local support staff</p>
```

Comment:
Shouldn't we include a "standard" footer (maybe the one we already have but more abstracted) to provide support information that can be customized by clients (email, phone number, ip address, mac address) ?